handler_success.go

```diff
@@ -43,6 +45,21 @@ type SuccessHandler struct {
 	BaseMiddleware
 }
 
+func tagHeaders(r *http.Request, th map[string]interface{}, tags []string) []string {
+	for k, v := range r.Header {
+		cleanK := strings.ToLower(k)
```

Comment:
I would use `net/http.CanonicalHeaderKey` here. Unless we need lowercase for a reason in particular?

In fact, the map is already using `CanonicalHeaderKey` internally, so you don't need to convert it at all.

Reply:
Whatever we go for - canonical keys or lowercase keys - we should error if the user gives us a `TagHeaders` that contains keys that don't comply to the format. Or convert them ourselves.

Reply:
I'll force lowercase, I don't want to leave it up to the user and converting to CanonicalHeaderKey seems unnecessary.